micros/modules/networking/networking.nix

```
@@ -181,7 +181,9 @@ in {
         hosts:     files dns
         networks:  files dns
       '';
+      "hosts".text = ''
 
+      '';
```

```suggestion
      hosts.text = "";
```

attrset keys don't need to be quoted unnless they begin with numbers or weird chars (or are nested paths)